homotopy-graphics/src/tikz.rs

```diff
@@ -97,10 +141,18 @@ pub fn render(diagram: &Diagram, stylesheet: &str) -> Result<String, DimensionEr
     }
 
     // Points
+    // TODO(thud): this `default_shape` should not be hardcoded here
```

Comment:
You can get rid of the default shape if `generator_styles` isn't an `Option`.

Reply:
You can just `unwrap` here. If the generator isn't in the signature, something has gone wrong.

Reply:
Great, that's sorted now. There was something similar in the manim rendering engine which I have now also fixed.